### DISCUSSION TITLE
All minor version of packages will stay behind one version

### DISCUSSION THREAD

**Discussion by josephphyo** [renovatebot/renovate#29911] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hello, 

How could i configure renovate for every minor versions of packages will stay behind one version from latest version? I tried with "allowedVersions" with version constraints but it can be specific package manager (eg, argocd,terraform) and not for every packages.

Thank you  

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
You mean you got the logic working using allowedVersions ie. PRs are now created for (latest.minor - 1). But now it is applied to all packages and you wish to apply it only for package of selected managers?

**Reply by josephphyo** [reply]
No , I would like to make all packages minor version will stay one version behind update compared with official latest release. not specific packages. 

For example, but all minor version are goes to latest release.

`"packageRules": [
    {
      "groupName": "All Minor Updates Stay One Version Behind",
      "groupSlug": "update-minor-version-behind",
      "labels": [
        "UPDATE-MINOR"
      ],
      "matchUpdateTypes": [
        "minor"
      ],
      "allowedVersions": "<{currentVersion.major}.{currentVersion.minor}"
    }
  ]`

**Reply by RahulGautamSingh** [reply, marked as answer]
Not possible.
